From cda1734bed3b048c01452c798877d05b8c2f4c15 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 6 Sep 2022 10:00:59 -0700
Subject: [PATCH 2/2] builtin: Replace _BSD_SOURCE with _DEFAULT_SOURCE

newer glibc has remove _BSD_SOURCE and wants it to be replaced with _DEFAULT_SOURCE

Fixes
/usr/include/features.h:194:3: warning: "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-W#warnings]
warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"

Uptream-Status: Submitted [https://github.com/stedolan/jq/pull/2480]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/builtin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/builtin.c b/src/builtin.c
index 1c6b08c..2a31496 100644
--- a/src/builtin.c
+++ b/src/builtin.c
@@ -1,4 +1,4 @@
-#define _BSD_SOURCE
+#define _DEFAULT_SOURCE
 #define _GNU_SOURCE
 #ifndef __sun__
 # define _XOPEN_SOURCE
-- 
2.37.3

